tsconfig.json

```diff
@@ -4,22 +4,14 @@
     "target": "es5",
     "moduleResolution": "node",
     "noImplicitAny": true,
-    "lib": [
-      "es6"
-    ],
+    "lib": ["es6"],
     "rootDir": "src",
     "declaration": true,
     "allowSyntheticDefaultImports": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "strict": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": true
```

Comment:
I don't know why they removed the last comma here (which I think it should be there), but in other examples, added the last comma. Such as in "src/models/Subscription.ts" they added a comma after `Ephemeral` 🤷🏻‍♀️